src/Common/src/Interop/Windows/sspicli/SSPIAuthType.cs

```diff
@@ -45,19 +45,14 @@ public int AcquireCredentialsHandle(string moduleName, Interop.SspiCli.Credentia
             return SafeFreeCredentials.AcquireCredentialsHandle(moduleName, usage, ref authdata, out outCredential);
         }
 
-        public int AcceptSecurityContext(SafeFreeCredentials credential, ref SafeDeleteContext context, SecurityBuffer[] inputBuffers, Interop.SspiCli.ContextFlags inFlags, Interop.SspiCli.Endianness endianness, SecurityBuffer outputBuffer, ref Interop.SspiCli.ContextFlags outFlags)
+        public int AcceptSecurityContext(SafeFreeCredentials credential, ref SafeDeleteContext context, Span<SecurityBuffer> inputBuffers, Interop.SspiCli.ContextFlags inFlags, Interop.SspiCli.Endianness endianness, SecurityBuffer outputBuffer, ref Interop.SspiCli.ContextFlags outFlags)
```

Comment:
Nit: Should any of these Span<SecurityBuffer> usages be ReadOnlySpan<SecurityBuffer>? I suspect it doesn't matter from a functional point of view because we're mostly just taking the address of these structures. But from a code readability point of view, if these things are not modified, it would be nice to make them ReadOnlySpan.

Reply:
Sure, I'll audit them to switch any over I can.